common/gratia/common/Gratia.py

```diff
@@ -381,6 +381,23 @@ def Processors(
         self.AppendToList(self.RecordData, 'Processors', pstring + self.Metric(metric)
                           + self.Description(description), str(value))
 
+    def GPUs(
+        self,
+        value,
+        consumptionRate=0,
+        metric='total',
+        description=r'',
+        ):
+        """ Metric should be one of 'total','average','max','min'
+        """
+
+        if consumptionRate > 0:
+            pstring = 'consumptionRate="' + str(consumptionRate) + '" '
```

Comment:
Just to double check - is it correct for this `pstring` part to be just `consumptionRate=...` instead of being prefixed like `urwg:metric=`, `urwg:description=`, etc ?